### PR TITLE
add JuliaAstro project

### PR DESCRIPTION
**Pull request by mileslucas**

### CODE REVIEW

File: _projects/2020/juliaastro/ccd_data_reduction.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# Student requirements:
+ - Basic knowledge of CCD data
+ - Familiarity with Julia and Julia packaging
+difficulty: low
+issues:
+# Related issues (if any)  to this project.
+mentors:
+# First person in contact; mentors may change before project starts.
+# GitHub handles
+ - mileslucas
```
Comment (bsipocz) [inline _projects/2020/juliaastro/ccd_data_reduction.md]: We have a requirement to have at least two mentors per projects. Could you please add someone else agree to co-mentor?

Comment (mileslucas) [reply _projects/2020/juliaastro/ccd_data_reduction.md]: no problem- I'll find another

**Comment by bsipocz** [comment]
@giordano and @yashrsharma44 - please confirm you're OK with mentoring gsoc this summer. Then I'll merge this PR.

**Comment by giordano** [comment]
Yes, I confirm I'm happy to co-mentor.

**Comment by yashrsharma44** [comment]
Yes, I am excited to work as a co-mentor!

**Pull request merged**